hackage-security/hackage-security.cabal

```diff
@@ -99,8 +99,8 @@ library
                        base16-bytestring >= 0.1.1   && < 0.2,
                        base64-bytestring >= 1.0     && < 1.1,
                        bytestring        >= 0.9     && < 0.11,
-                       Cabal             >= 1.14    && < 1.26,
```

Comment:
This line here makes me suspect this is against some fairly ancient state of the master branch